src/urbit/app/chess.hoon

```diff
@@ -1125,12 +1125,13 @@
         (check-50-move-rule u.new-position)
```

Comment:
Don't forget that you'll also need to update the `%position` cards sent on line `818`. It's possible to do so without having to touch the type of `moves.game.game-state`, but you'll need to use `spin` (or possibly `spun`) instead of `turn` (let me know if that's not enough of a directional hint).